Scripts/Mac/CleanupFunctionalTests.sh

```diff
@@ -3,6 +3,7 @@
 pkill -9 -l GVFS.FunctionalTests
 pkill -9 -l git
 pkill -9 -l gvfs
+pkill -9 -l GVFS.Mount
```

Comment:
This is an unrelated change - Functional tests were failing when a previous test was cancelled.